src/modules/generators/cylinders.rs

```diff
@@ -0,0 +1,40 @@
+// Copyright 2015 The Noise-rs Developers.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use num::Float;
+use {math, lib};
+
+pub struct Cylinders<T> {
+	frequency: T,
+}
```

Comment:
Not really, the implementation creates concentric cylinders, not a single cylinder.